3rdparty/openvx/include/ivx.hpp

```diff
@@ -176,6 +176,27 @@ template<> struct TypeToEnum<vx_bool>     { static const vx_enum value = VX_TYPE
 //template<> struct TypeToEnum<vx_size>     { static const vx_enum val = VX_TYPE_SIZE; };
 //template<> struct TypeToEnum<vx_df_image> { static const vx_enum val = VX_TYPE_DF_IMAGE; };
 
+#ifdef IVX_USE_OPENCV
+inline int enumToCVType(vx_enum type)
+{
+    switch (type)
+    {
+    case VX_TYPE_CHAR: return CV_8SC1;
```

Comment:
while it's theoretically correct, I would suggest use of `CV_8U` here since `CV_8S` support in OpenCV is incomplete...